### PR TITLE
[TDV-452] Feature/show limits request details

### PR DESCRIPTION
**Pull request by Datikzz**

### CODE REVIEW

File: src/vue/pages/Limits/Limits.RequestDetailsViewer.vue [hunk 1]
```diff
@@ -0,0 +1,202 @@
+<template>
+  <div class="limits-request-details-viewer">
+    <div
+      v-if="request.isApproved"
+      class="request-state request-state--approved"
```
Comment (max-nes) [inline src/vue/pages/Limits/Limits.RequestDetailsViewer.vue]: `limits-request-details-viewer__request-state limits-request-details-viewer__request-state--approved`

File: src/vue/pages/Limits/Limits.RequestDetailsViewer.vue [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+    >
+      <p class="request-state__content">
```
Comment (max-nes) [inline src/vue/pages/Limits/Limits.RequestDetailsViewer.vue]: `limits-request-details-viewer__request-state-content`

File: src/vue/pages/Limits/Limits.RequestDetailsViewer.vue [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+      </p>
+    </div>
+
+    <div
+      v-else-if="request.isRejected"
+      class="request-state request-state--rejected"
+    >
+      <p class="request-state__content">
+        <!-- eslint-disable-next-line max-len -->
+        {{ 'limits-request-details-viewer.rejected-msg' | globalize() }}
```
Comment (max-nes) [inline src/vue/pages/Limits/Limits.RequestDetailsViewer.vue]: `globalize()` => `globalize`

File: src/vue/pages/Limits/Limits.RequestDetailsViewer.vue [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+    <div
+      v-else-if="request.isPermanentlyRejected"
+      class="request-state request-state--permanently-rejected"
+    >
+      <p class="request-state__content">
+        <!-- eslint-disable-next-line max-len -->
+        {{ 'request-messages.permanently-rejected-msg' | globalize({ reason: request.rejectReason }) }}
+      </p>
+    </div>
+    <div class="app__table limits-request-details__table">
```
Comment (max-nes) [inline src/vue/pages/Limits/Limits.RequestDetailsViewer.vue]: `limits-request-details__table` => `limits-request-details-viewer__table`

pls recheck that all the other classes are compliant with BEM

File: src/vue/pages/Limits/Limits.RequestDetailsViewer.vue [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+<script>
+
+import { LimitsUpdateRequestRecord } from '@/js/records/requests/limits-update.record'
+
+import {
+  LIMITS_REQUEST_STATES_STR,
+  LIMITS_REQUEST_TYPE,
+} from '@/js/const/limits.const'
+
+const LIMITS_REQUEST_STATES_STR_TRANSLATE_ID = Object.freeze({
```
Comment (max-nes) [inline src/vue/pages/Limits/Limits.RequestDetailsViewer.vue]: `LIMITS_REQUEST_STATES_STR_TRANSLATION_ID`

File: src/vue/pages/Limits/Limits.RequestDetailsViewer.vue [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+  [LIMITS_REQUEST_STATES_STR.pending]: 'limits-requests-table-renderer.request-state-pending',
+  [LIMITS_REQUEST_STATES_STR.cancelled]: 'limits-requests-table-renderer.request-state-cancelled',
+  [LIMITS_REQUEST_STATES_STR.approved]: 'limits-requests-table-renderer.request-state-approved',
+  [LIMITS_REQUEST_STATES_STR.rejected]: 'limits-requests-table-renderer.request-state-rejected',
+  [LIMITS_REQUEST_STATES_STR.permanentlyRejected]: 'limits-requests-table-renderer.request-state-permanently-rejected',
+})
+
+const LIMITS_REQUEST_TYPE_TRANSLATE_ID = Object.freeze({
```
Comment (max-nes) [inline src/vue/pages/Limits/Limits.RequestDetailsViewer.vue]: `LIMITS_REQUEST_TYPE_TRANSLATION_ID`

File: src/vue/pages/Limits/Limits.RequestsListRenderer.vue [hunk 7]
```diff
@@ -98,8 +97,17 @@
...
+    </drawer>
+
+    <drawer :is-shown.sync="isDetailsDrawerShown">
+      <template slot="heading">
+        <!-- eslint-disable-next-line -->
+        {{ 'limits-requests-table-renderer.request-details-drawer-heading' | globalize }}
+      </template>
+      <limits-request-details-viewer
+        @upload="showDocumentUploadForm"
+        :request="selectedRequest" />
```
Comment (max-nes) [inline src/vue/pages/Limits/Limits.RequestsListRenderer.vue]: `/>` should be on new line

Comment (Datikzz) [reply src/vue/pages/Limits/Limits.RequestsListRenderer.vue]: Edited

File: src/vue/pages/Limits/Limits.RequestDetailsViewer.vue (hunk 6 above)
Comment (Datikzz) [reply src/vue/pages/Limits/Limits.RequestDetailsViewer.vue]: Renamed

File: src/vue/pages/Limits/Limits.RequestDetailsViewer.vue (hunk 5 above)
Comment (Datikzz) [reply src/vue/pages/Limits/Limits.RequestDetailsViewer.vue]: Renamed

File: src/vue/pages/Limits/Limits.RequestDetailsViewer.vue (hunk 3 above)
Comment (Datikzz) [reply src/vue/pages/Limits/Limits.RequestDetailsViewer.vue]: Edited

File: src/vue/pages/Limits/Limits.RequestDetailsViewer.vue (hunk 2 above)
Comment (Datikzz) [reply src/vue/pages/Limits/Limits.RequestDetailsViewer.vue]: Edited

File: src/vue/pages/Limits/Limits.RequestDetailsViewer.vue (hunk 1 above)
Comment (Datikzz) [reply src/vue/pages/Limits/Limits.RequestDetailsViewer.vue]: Edited

File: src/vue/pages/Limits/Limits.RequestDetailsViewer.vue (hunk 4 above)
Comment (Datikzz) [reply src/vue/pages/Limits/Limits.RequestDetailsViewer.vue]: Fixed

File: src/vue/pages/Limits/Limits.RequestDetailsViewer.vue (hunk 6 above)
Comment (max-nes) [reply src/vue/pages/Limits/Limits.RequestDetailsViewer.vue]: Still not renamed in the current file

File: src/vue/pages/Limits/Limits.RequestDetailsViewer.vue (hunk 5 above)
Comment (max-nes) [reply src/vue/pages/Limits/Limits.RequestDetailsViewer.vue]: 1. still not renamed in the current file
2. looks like it is unused in the current file

Comment (Datikzz) [reply src/vue/pages/Limits/Limits.RequestDetailsViewer.vue]: My apologies.

**Pull request merged**